test/documentation/seqan3.css

```diff
@@ -0,0 +1,111 @@
+body {
+        background-color: #CCC;
+        color: black;
+        font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+
+}
+
+div, p, dl {
+        font-size: 13pt;
+        font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+        line-height: 30px;
+}
+
+table {
+        font-size: 12pt;
+        font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+
+}
+
+div.contents {
+        margin-bottom: 10px;
+        padding: 40px;
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 960px;
+        background-color: white;
+        border-radius: 8px;
+}
+
+#titlearea {
+        background-color: white;
+        align-content: center;
+}
+
+hr.footer {
+        display: none;
+}
+
+.title {
+    text-align: center;
+}
+
+.footer {
+        background-color: #AAA;
+}
+
+div.fragment {
+        font-size: 12pt;
+        line-height: 30px;
+        font-family: ubuntu-mono, sans-serif;
+        padding: 15px;
+        border-radius: 8px;
+        background-color: #F2F2F2;
+        border: 1px solid #F2F2F2;
+}
+
+code {
+        font-size: 12pt;
+        line-height: 15px;
+        font-family: ubuntu-mono, sans-serif;
+        padding: 2px;
+        border-radius: 1px;
+        background-color: #F2F2F2;
+        border: 1px solid #F2F2F2;
+}
+
+dl.section {
+    padding: 10px;
+}
+
+dl.attention {
+    background-color: #fff1f1;
+}
+
+dl.warning {
+    border-color: #E08D12;
+    background-color: #FBF0E0;
+}
+
+dl.note {
+    border-color: #E9DF24;
+    background-color: #FCFBE6;
+}
+
+hr {
+    border-top: 2px solid #E4E4E4;
+    height: 2px;
+}
+
+table.doxtable{empty-cells:show;border:1px solid #cbcbcb}
```

Comment:
sometimes you use `key:value` and sometimes `key: value` (with a space)

Reply:
And I think it is more readable if you would break down those one-liners, i.e., each key-value pair on a separate line.